### ISSUE TITLE
Vulnerable driver blocklist

### ISSUE THREAD

**Issue by theo-stack** [cheat-engine/cheat-engine#2485]
Hello there, it seems like i can't set this "VulnerableDriverBlocklistEnable" to "0" in the regedit is there a reason why ?
I followed the steps but i can't disable it.
If anyone can help i'll be grateful.

**Comment by fade2gray** [comment]
Have you tried running the registry editor as admin?

**Comment by theo-stack** [comment]
> Have you tried running the registry editor as admin?

Yes always

**Comment by fade2gray** [comment]
So, you followed the steps similar to those shown [here](https://github.com/cheat-engine/cheat-engine/issues/2461#issuecomment-1385043409)  - what error are you getting?

**Comment by cheat-engine** [comment]
Why can't you change it?
Does it return back to 1, or is it because it doesn't exist in the list?
If it doesn't exist,  create it, and then set it to 0.  KJeep in mind that it has to be a `value`, with that name, not a `key`

**Comment by theo-stack** [comment]
> Why can't you change it? Does it return back to 1, or is it because it doesn't exist in the list? If it doesn't exist, create it, and then set it to 0. Keep in mind that it has to be a `value`, with that name, not a `key`

i'm may be dumb sorry lemme try that

**Comment by cheat-engine** [comment]
and did you restart windows ? (So not shut down and back on, actual restart?)

**Comment by theo-stack** [comment]
> and did you restart windows ? (So not shut down and back on, actual restart?)

yes restart not shut down and power back on, i did a complete restart

**Comment by theo-stack** [comment]
i did something like that
![image](https://user-images.githubusercontent.com/73785792/219378540-85692970-5d75-4641-92f4-860c9290b772.png)

**Comment by cheat-engine** [comment]
Assuming it's in the correct location ( HKEY_LOCAL_MACHINE\SYSTEM\CurrentControlSet\Control\CI\Config ) , then after a full restart that normally should work.  

if not, perhaps there';s another AV giving the exact same errorcode

**Comment by theo-stack** [comment]
yes its in this path, and i don't have an other AV except for win defender

**Comment by cheat-engine** [comment]
try https://support.microsoft.com/en-au/topic/kb5020779-the-vulnerable-driver-blocklist-after-the-october-2022-preview-release-3fcbe13a-6013-4118-b584-fcfbc6a09936

**Comment by theo-stack** [comment]
![image](https://user-images.githubusercontent.com/73785792/219382985-c716f9d3-0b6a-4cf3-b42b-b6aa1c4ef480.png)
it's disabled

**Comment by theo-stack** [comment]
i don't have the windows in S mode even tho i did what they said

**Comment by cheat-engine** [comment]
If nothing works (and the restarting of windows actually happens and isn't faked by windows)  then try disabling secure boot in your BIOS

**Comment by theo-stack** [comment]
Still not working sadly,
I wonder what's the problem

**Comment by theo-stack** [comment]
> If nothing works (and the restarting of windows actually happens and isn't faked by windows) then try disabling secure boot in your BIOS

I tried searching for this issue but i can't find a fix, that's weird

**Comment by cheat-engine** [comment]
i have no idea. I have tested on multiple systems including the latest win11 update and there i fo get it to work.

Is this a desktop or laptop?  If desktop, have you tried pulling the power plug and  then hold the power button for a few seconds ? and then back on again ?

**Comment by theo-stack** [comment]
> i have no idea. I have tested on multiple systems including the latest win11 update and there i do get it to work.
> 
> Is this a desktop or laptop? If desktop, have you tried pulling the power plug and then hold the power button for a few seconds ? and then back on again ?

Let me try that then

**Comment by theo-stack** [comment]
oh

**Comment by theo-stack** [comment]
![image](https://user-images.githubusercontent.com/73785792/219824497-8ce2df91-cce9-4f4a-b652-6b2085ceebff.png)

**Comment by theo-stack** [comment]
> i have no idea. I have tested on multiple systems including the latest win11 update and there i do get it to work.
> 
> Is this a desktop or laptop? If desktop, have you tried pulling the power plug and then hold the power button for a few seconds ? and then back on again ?

fixed

**Comment by KDot227** [comment]
@cheat-engine I'm having the same issue

![image](https://user-images.githubusercontent.com/110929758/231560424-0c6e1a61-ea8e-4566-9d3c-6971b9c600f3.png)

**Comment by KDot227** [comment]
I tried all the steps and nothing seems to be working either
